meetings/approvals/tests.py

```diff
@@ -1,12 +0,0 @@
-from django.test import TestCase
-from unittest import skip
-
-
-class TestApprovals(TestCase):
```

Comment:
Oh my bad, I actually thought this was a different chunk of code that I removed. I'll put it back.